docs/tests/integration/post_v1_message-batches/validation.md

```diff
@@ -193,6 +193,69 @@ This test uses the ‘X-Correlation-Id’ header, when provided in a request it
 | 76491414-d0cf-4655-ae20-a4d1368472f3 | Is tested to ensure that when a correlation identifier is sent, we respond with the same value.               |
 
 
+## Scenario: An API consumer submitting a request with an invalid address postcode receives a 400 ‘Invalid Value’ response
+
+A valid contact detail must be structured in this format: { sms: value, email: value, address: { lines: [], postcode: value } }
+
+**Given** the API consumer provides an message body with an invalid postcode
+<br/>
+**When** the request is submitted
+<br/>
+**Then** the response returns a 400 invalid value error
+<br/>
+
+**Asserts**
+- Response returns a 400 ‘Invalid Value’ error
+- Response returns the expected error message body with references to the invalid attribute
+- Response returns the ‘X-Correlation-Id’ header if provided
+
+| Value     | Description                                    |
+|-----------|------------------------------------------------|
+| LS1 6AECD | Used to ensure only valid postcode is accepted |
+
+
+## Scenario: An API consumer submitting a request with invalid address lines (too few) receives a 400 ‘Missing Value’ response
```

Comment:
This will change with @Ian-Hodges's ticket [CCM-6471](https://nhsd-jira.digital.nhs.uk/browse/CCM-6471) although probably not worth changing now.